src/Uno.UI.BindingHelper.Android/Uno/UI/UnoMotionHelper.java

```diff
@@ -181,6 +181,14 @@ private boolean dispatchMotionEventCore(Uno.UI.MotionTargetAdapter adapter, Moti
 		if (adapter.getCustomDispatchIsActive()) {
 			// Log.i(LOGTAG, _indent + "CUSTOM dispatch (" + target.getChildrenRenderTransformCount() + " of " + view.getChildCount() + " children are transformed )");
 
+			// Because we do not call dispatchToSuper(), we must manually call requestDisallowInterceptTouchEvent in the same circumstances
```

Comment:
A+ comment 👍

Reply:
Actually I don't understand what the purpose of this change ...

First I think that the `view.requestDisallowInterceptTouchEvent` should have been handled at [line 148 - 151](https://github.com/unoplatform/uno/pull/2338/files#diff-365e56e68abf23c7d1195c0de11ab3a9R148) in order to be consistent

But the main issue for me is that from what I understand here is that as soon as we have an element with a `RenderTransform` it will prevent any parent `ScrollViewer` to do it job. It seems a bit restrictive ...

@jcantin-nventive

Reply:
Actually as it set it to `false` it's the opposite, so it means that it breaks the manipulation events when in a scroll viewer.

Reply:
@dr1rrb It's just replicating the behaviour of the `super` method, specifically this line: https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/view/ViewGroup.java;l=2839?q=resettouchstate

Reply:
Ok that gives more context. 

So in my point of view, for the _down_ ([line 2600 of the ViewGroup](https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/view/ViewGroup.java;bpv=1;bpt=1;l=2600)) it means that our [lines 148-151](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.BindingHelper.Android/Uno/UI/UnoMotionHelper.java#L148) are not at the right place and should be moved **after** the super dispatch as `super` will erase what just did! (Basically I would do the exact opposite of this change: make sure that the `view.requestDisallowInterceptTouchEvent(true)` is properly set.) 

I would also agree to force a reset for the _up_, _cancel_ and _move over_ (even if I would argue about the utility for the last one) after a "custom dispatch".

And reading this, I realized that there is a **major** difference between the `resetTouchState` and the `view.requestDisallowInterceptTouchEvent`: the first one is only for safety and reset a **local** flag, while the second will bubble up in the whole visual tree. I'm pretty sure the real issue is here: we request a disallow on a control on line 151, then invoke the super which reset only the local flag (ViewGroup@2600) which drives us into an invalid state which cannot be recovered properly.

Reply:
And actually if we look at the implementation of the [`requestDisallowInterceptTouchEvent` in the `ViewGroup`](https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/view/ViewGroup.java;l=3168), it does not propagates the request if the **local flag** is not set ... it really looks like the described issue.

Reply:
FYI I fixed this in #2445 I've validated your test, and it seems to be OK, but let me know if you have any issue with the updated fix.